docs/wiki/Known issues.md

```diff
@@ -1,34 +1,39 @@
-# Known issues
-
 This section provides an overview of the most impactful limitations and known issues. We are actively working on tracking them as GitHub issues and resolving them.
 
 ---
 
 ### _Navigation_
 
 - [Module specific](#module-specific)
+  - [Microsoft.AAD/DomainServices](#microsoftaaddomainservices)
+  - [Microsoft.KubernetesConfiguration/extensions](#microsoftkubernetesconfigurationextensions)
+  - [Microsoft.KubernetesConfiguration/fluxConfigurations](#microsoftkubernetesconfigurationfluxconfigurations)
+  - [Microsoft.Network/vpnGateways](#microsoftnetworkvpngateways)
+  - [Microsoft.Network/virtualHubs](#microsoftnetworkvirtualhubs)
+  - [Microsoft.Network/vpnSites](#microsoftnetworkvpnsites)
```

Comment:
Just a heads-up for pulling latest changes as we just merged an update to this page.